components/homme/src/theta-l/model_init_mod.F90

```diff
@@ -92,7 +92,8 @@ subroutine model_init2(elem,hybrid,deriv,hvcoord,tl,nets,nete )
        ! active for p<10*ptop (following cd_core.F90 in CAM-FV)
        ! CAM 26L and 30L:  top 3 levels 
        ! E3SM 72L:  top 6 levels
-       nu_scale_top(k) = 8*(1+tanh(log(ptop_over_press))) ! active for p<4*ptop
+       nu_scale_top(k) = 16*ptop_over_press**2 / (ptop_over_press**2 + 1)
+
```

Comment:
can you leave the original formula in the code, commented out?  So it is clear that it is the formula taken from CAM-FV.

Reply:
Done, thanks.